### ISSUE TITLE
Saving Progress

### ISSUE THREAD

**Issue by acocalypso** [szmyty/OpenAI-Retro-SuperMarioWorld-SNES#4]
Hi,

is there any option to save the progress after each generation?
I would like to stop it and start it during the day.

**Comment by Ploita** [comment]
Yes, just need to add some code, use `checkpoint_file = 'neat-checkpoint-XXX'`, to load a checkpoint, change XXX for the version you want to load, and switch `p = neat.Population(config)` for `p = neat.Checkpointer.restore_checkpoint(checkpoint_file)` to continue your progress
